ansible/roles/baselayout/tasks/main.yml

```diff
@@ -66,8 +66,8 @@
   stat: path="{{ binary_dest }}"
   register: has_git
 
-- name: centos | remove existing git package
-  when: "(os == 'centos6' or os == 'centos7') and arch != 'arm64'"
+- name: centos6 | remove existing git package
+  when: "(os == 'centos6') and arch != 'arm64'"
```

Comment:
AFAICT, this doesn't need to be done anymore. cc: @rvagg

Reply:
The brackets can be dropped here now the "or" clause is gone.